src/lib/components/VideoCard/VideoCard.svelte

```diff
@@ -1,78 +1,154 @@
 <script lang="ts">
   import "./VideoCard.scss";
 
-  import { afterUpdate } from "svelte";
-
-  import getYoutubeVideoData from "$lib/services/youtube";
+  import type { Thumbnails } from "$lib/services/server/youtube";
+  import getYoutubeVideoIDFromURL from "$lib/util/getYoutubeVideoIDFromURL";
   import YoutubeSubscribeLink from "$lib/components/YoutubeSubscribeLink";
+  import Image, { type Sources, type Srcset } from "$lib/components/Image";
 
   type VideoCardVariation = "hero" | "primary" | "secondary" | "tertiary";
   interface $$Props {
     class?: string;
     url: string;
-    customTitle?: string | null | undefined;
-    customDescription?: string | null | undefined;
+    title?: string | null | undefined;
+    description?: string | null | undefined;
     variation?: VideoCardVariation;
+    blurhash?: string | null | undefined;
+    thumbnails?: Thumbnails;
+    sizeType?: SizeType;
   }
 
   let className = "";
   export { className as class };
   export let url = "";
-  export let customTitle: $$Props["customTitle"] = null;
-  export let customDescription: $$Props["customDescription"] = null;
+  export let title: $$Props["title"] = null;
+  export let description: $$Props["description"] = null;
   export let variation = "primary";
 
-  let youtubeVideoId: string | null;
+  const maxDescriptionLength = 500;
+  $: trimmedFirstParagraph = description
+    ?.replace(/\n[\s\S]*$/, " ") // remove everything after the first newline
+    .slice(0, maxDescriptionLength) // limit to maxDescriptionLength
+    .replace(/\s[^\s]*?$/, ""); // remove everything after the start of the last whitespace character
```

Comment:
This is removing the last word on the following pages:
* [/plants](https://ldaf-git-benaiah-dont-load-youtube-embed-until-its-clicked-ldaf.vercel.app/plants)
* [/land](https://ldaf-git-benaiah-dont-load-youtube-embed-until-its-clicked-ldaf.vercel.app/land)
* [/business](https://ldaf-git-benaiah-dont-load-youtube-embed-until-its-clicked-ldaf.vercel.app/business)

Reply:
good catch, should be fixed now